src/ngx_http_lua_socket_tcp.c

```diff
@@ -5332,15 +5862,21 @@ ngx_http_lua_socket_tcp_resume_helper(ngx_http_request_t *r, int socket_op)
 
     dd("coctx: %p", coctx);
 
-    u = coctx->data;
-
     switch (socket_op) {
+    case SOCKET_OP_RESUME_CONN:
```

Comment:
Please add a blank line before the first `case` statement to better follow the nginx coding style. I understand the original code seems to already have the same problem, but it's good to fix things when we touch the same part of the code.